tests/scores/test_continuous.py

```diff
@@ -98,6 +101,100 @@ def test_2d_xarray_mse_with_dimensions():
     assert all(result.round(4) == expected_values)
     assert result.dims == expected_dimensions
 
+@pytest.fixture
+def rmse_fcst_xarray():
+    return xr.DataArray([1, 3, 1, 3, 2, 2, 2, 1, 1, 2, 3])
+
+@pytest.fixture
+def rmse_obs_xarray():
+    return xr.DataArray([1, 1, 1, 2, 1, 2, 1, 1, 1, 3, 1])
+
+@pytest.fixture
+def rmse_fcst_pandas():
+    return pd.Series([1, 3, 1, 3, 2, 2, 2, 1, 1, 2, 3])
+
+@pytest.fixture
+def rmse_obs_pandas():
+    return pd.Series([1, 1, 1, 2, 1, 2, 1, 1, 1, 3, 1])
+
+def test_rmsd_xarray_1d(rmse_fcst_xarray, rmse_obs_xarray):
+    """
+    Test basic existence of rmsd
+    """
+    result = scores.continuous.rmsd(rmse_fcst_xarray, rmse_obs_xarray)
+
+    expected = xr.DataArray(1.0445)
+    assert result.round(PRECISION) == expected.round(PRECISION)
+
+@pytest.mark.parametrize(
+    "forecast, observations, expected, request_kwargs", 
+    [
+        ('rmse_fcst_xarray', 'rmse_obs_xarray', xr.DataArray(1.0445), {}),
+        ('rmse_fcst_xarray', 'rmse_obs_xarray', xr.DataArray([0, 2, 0, 1, 1, 0, 1, 0, 0, 1, 2]), dict(preserve_dims = 'all')),
+        ('rmse_fcst_xarray', 1, xr.DataArray(1.206), {}),
+        ('rmse_fcst_pandas', 'rmse_obs_pandas', 1.0445, {}),
+    ],
+    ids=["simple-1d", "preserve-1d", "to-point","pandas-series-1d"]
+)
+def test_rmse_xarray_1d(forecast, observations, expected, request_kwargs, request):
+    """
+    Test RMSE for the following cases:
+       * Calculates the correct value for a simple xarray 1d sequence
+       * Calculates the correct value for an xarray 1d sequence preserving all
+       * Calculates the correct value for a simple pandas 1d series 
+       * Calculates the correct value for an xarray 1d sequence comparing to a point
+    """
+    if isinstance(forecast, str): forecast = request.getfixturevalue(forecast)
+    if isinstance(observations, str): observations = request.getfixturevalue(observations)
+    result = scores.continuous.rmse(forecast, observations, **request_kwargs)
+
+    assert result.round(PRECISION) == expected
+
+
+
+@pytest.fixture
+def rmse_2d_fcst_xarray():
+    numpy.random.seed(0)
+    lats = [50, 51, 52, 53]
+    lons = [30, 31, 32, 33]
+    fcst_temperatures_2d = 15 + 8 * np.random.randn(1, 4, 4)
```

Comment:
For the tests, can you add a NaN in the data in ` rmse_2d_fcst_xarray` and `rmse_2d_obs_xarray` to check that it behaves as expected.
I also suggest that for future tests, you don't generate the input data randomly so that you can try and capture edge cases (e.g., negative numbers or 0's) . It also helps with calculating the expected output independently with carefully chosen test inputs for more complicated metrics. You wouldn't have to worry about rounding the output either in many cases when you have carefully chosen the input values.